src/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/Helpers.cs

```diff
@@ -14,6 +14,11 @@ public static byte[] CloneByteArray(this byte[] src)
             return (byte[])(src.Clone());
         }
 
+        public static KeySizes[] CloneKeySizesArray(this KeySizes[] src)
+        {
+            return (KeySizes[])(src.Clone());
+        }
+
```

Comment:
Should we just have a helper:
```C#
public static T[] CloneArray<T>(this T[] src)
{
    return (T[])src.Clone();
}
```
rather than one method per T?